### PR TITLE
Remove undeclared properties from SelfSVC form

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Trying to fish out my old work in the context of https://github.com/civicrm/civicrm-core/pull/27769

Before
----------------------------------------
_What is the old user-interface or technical-contract (as appropriate)?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

After
----------------------------------------
_What changed? What is new old user-interface or technical-contract?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

Technical Details
----------------------------------------
_If the PR involves technical details/changes/considerations which would not be manifest to a casual developer skimming the above sections, please describe the details here._

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

**Comment by colemanw** [comment]
@jofranz have you tested this PR? It would be helpful if you could say whether this should be merged.

File: CRM/Event/Form/SelfSvcUpdate.php [hunk 1]
```diff
@@ -123,8 +115,7 @@ public function preProcess() {
     $this->_event_id = $this->_part_values['event_id'];
     $url = CRM_Utils_System::url('civicrm/event/info', "reset=1&id={$this->_event_id}");
     $this->_contact_id = $this->_part_values['participant_contact_id'];
-    $validUser = CRM_Contact_BAO_Contact_Utils::validChecksum($this->_contact_id, $this->_userChecksum);
-    if (!$validUser && !CRM_Core_Permission::check('edit all events')) {
+    if (!$this->getAuthenticatedCheckSumContactID() && !CRM_Core_Permission::check('edit all events')) {
       CRM_Core_Error::statusBounce(ts('You do not have sufficient permission to transfer/cancel this participant.'), $url);
```
Comment (demeritcowboy) [inline CRM/Event/Form/SelfSvcUpdate.php:119]: I'm getting this bounce error and it works before the patch, so something is off with this refactor.

Comment (demeritcowboy) [reply CRM/Event/Form/SelfSvcUpdate.php:119]: getRequestedContactID() is returning NULL

Comment (demeritcowboy) [reply CRM/Event/Form/SelfSvcUpdate.php:119]: This PR has conflicts now too, so will close.

**Pull request closed without merging**